README.org

```diff
@@ -18,6 +18,15 @@ wget https://github.com/NASA-GISS/econ/archive/refs/heads/main.zip -O econ.zip
 unzip -j -o econ.zip econ-main/econ econ-main/econ_includes.sh
 #+END_SRC
 
+** Installation with git access
```

Comment:
Good, although it doesn't need to be in `exec` folder. You need to run it from `decks`, but from `decks` can call `/path/to/econ.git/econ`.

Reply:
True, but I thought it would be more logical to keep it in exec. As you wish, we could remove this section.

Reply:
I'm happy to keep it - just an FYI for your development purposes.